www/%username/widgets/%type.spt

```diff
@@ -2,15 +2,21 @@ from markupsafe import Markup
 
 from liberapay.utils import get_participant
 
-A, B = Markup('<br><span style="font-size: 125%">'), Markup('</span>')
+span_open, span_close, br = Markup('<br><span style="font-size: 125%">'), Markup('</span>'), Markup('<br>')
 
 [---]
 
 t = request.path['type']
 if t not in ('giving.js', 'receiving.js'):
-    raise response.error(400)
+    raise response.error(404)
 
 participant = get_participant(state, restrict=False, redirect_canon=False)
+goal = participant.goal
+show_goal = goal and goal > participant.receiving
+
+receiving_markup = "{0}{1}{2}{3}".format(span_open, participant.receiving, span_close, br)
+giving_markup = "{0}{1}{2}{3}".format(span_open, participant.giving, span_close, br)
+goal_markup = "{0}{1}{2}".format(span_open, goal, span_close)
```

Comment:
This isn't correct, it doesn't localize the money amounts. We should use `Wrap` from `liberapay.utils.i18n` instead.